Gopkg.lock

```diff
@@ -722,7 +733,7 @@
   analyzer-version = 1
   input-imports = [
     "github.com/apparentlymart/go-cidr/cidr",
-    "github.com/ghodss/yaml",
+    "github.com/coreos/go-semver",
```

Comment:
Add this pkg as constraint/override in Gopkg.toml?

Reply:
What is the reason to add it to `Gopkg.toml`? If we don't need a specific version, and if we're keeping the currently vendored version in Git, is there a reason to set a constraint?

Reply:
Added.